src/translations/sqlb_es_ES.ts

```diff
@@ -6020,7 +6023,7 @@ Please check again.</source>
     <message>
         <location filename="../RemoteDock.ui" line="86"/>
         <source>Upload</source>
-        <translation type="unfinished"></translation>
+        <translation>Upload</translation>
```

Comment:
El [glosario de KDE](https://es.l10n.kde.org/glosario.php) propone "cargar o enviar", por si lo queremos contemplar.

Reply:
No me he atrevido a traducirlo como Subir que es más claro pero depende del contexto.

No sé si Cargar será confuso y Enviar... Lo pongo como creas conveniente porque no he visto donde aparece y cómo va a quedar más claro.

Reply:
El contexto es este:
![imagen](https://github.com/sqlitebrowser/sqlitebrowser/assets/13812910/6e5a0519-2ca5-4564-aa02-535a7c5370e6)
Se me ocurre también publicar, aunque enviar tampoco me suena mal para eso. En 3.12 era un botón con icono y sin leyenda. El tooltip decía (y sigue diciendo) "Volcar la base de datos actualmente abierta al servidor". El original dice: "Push currently opened database to server". Así que en este caso tampoco es muy coherente en el original, porque la leyenda dice _upload_ y la ayuda _push_.

Reply:
Lo que yo veo es que la gente confunde mucho Cargar, Descargar y Recargar si no hay más contexto.

Como el botón tiene sitio para crecer "Cargar al servidor" podría ser más claro.
Y ya de paso cambiar también la traducción del tooltip para hacerlo coherente.

¿Cómo lo ves?

Reply:
No había visto esto. Pues sí, quedaría más claro, que es lo importante.

Reply:
Al final lo dejé como Cargar por no arriesgarme a hacer el botón demasiado grande y que pasase algo raro.

Si ves que queda bien de tamaño, lo puedes cambiar.

Reply:
Ok, cuando lo junte todo puedo echarle un ojo y cambiarlo si lo veo bien.